policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -429,3 +439,93 @@ impl ServiceRoutes {
         });
     }
 }
+
+// TODO: could bubble up errors through `Result`
+fn parse_accrual_config(
+    annotations: &std::collections::BTreeMap<String, String>,
+) -> Result<Option<FailureAccrual>> {
+    match annotations.get("balancer.linkerd.io/mode") {
+        Some(v) if v == "consecutive" => {
+            // TODO (matei): value is kind of out of pocket, how do we establish what's good
+            // here.
+            let max_failures = annotations
+                .get("balancer.linkerd.io/failure-accrual-consecutive-failures")
+                .map(|s| {
+                    s.parse::<u32>()
+                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
```

Comment:
this `map_err` is throwing away the error, since the `tracing::debug!` macro doesn't return anything. this function doesn't compile because it expects the function's error type to be `anyhow::Error`, but the error here is `()` (unit)